test/test_sendgrid.py

```diff
@@ -144,8 +144,11 @@ def test_hello_world(self):
         content = Content(
             "text/plain", "and easy to do anywhere, even with Python")
         mail = Mail(from_email, subject, to_email, content)
-        self.assertTrue(mail.get() == {'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}], 'personalizations': [
-                        {'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'}, 'subject': 'Sending with SendGrid is Fun'})
+        self.assertTrue(mail.get() == {
```

Comment:
```python
self.assertEqual(mail.get, {...})  # ?
```

Reply:
@42B 
I have updated file with master branch. This file was changed when merged with others PR.